docs-kits/kits/Industry Core Kit/Software Development View/page_aspect-models.mdx

```diff
@@ -491,103 +408,142 @@ The only special step in creating this aspect model is the initial creation of t
+    "BPNL50096894aNXY"
   ]
 }
 ```
 
-#### Aspect Model `SingleLevelBomAsBuilt 1.0.0` for a Batch
+
+
+#### Aspect Model `SingleLevelUsageAsBuilt 3.0.0` for a Batch
 
 ```json
 {
   "catenaXId": "urn:uuid:580d3adf-1981-44a0-a214-13d6ceed9379",
-  "childItems": [
+  "parentItems": [
     {
+      "catenaXId": "urn:uuid:055c1128-0375-47c8-98de-7cf802c3241d",
+      "isOnlyPotentialParent": false,
       "quantity": {
-        "quantityNumber": 25.0,
-        "measurementUnit": "unit:kilogram"
+        "value": 25.0,
+        "unit": "unit:kilogram"
       },
-      "createdOn": "2022-02-03T14:48:54.709Z",
-      "lastModifiedOn": "2022-02-03T14:48:54.709Z",
-      "catenaXId": "urn:uuid:d60b99b0-f269-42f5-94d0-64fe0946ed04",
-      "businessPartner": "BPNL50096894aNXY"
+      "createdOn": "2022-02-03",
+      "businessPartner": "BPNL50096894aNXY",
+      "lastModifiedOn": "2022-02-03T14:48:54.709Z"
     }
+  ],
+  "customers": [
+    "BPNL50096894aNXY"
   ]
 }
 ```
 
-#### Creation of Aspect Model SingleLevelBomAsBuilt
+#### Creation of Aspect Model SingleLevelUsageAsBuilt
 
-The creation of the aspect model SingleLevelBomAsBuilt is more complicated. This aspect model contains the Unique ID of the manufacturer's part (attribute catenaXId) which is created - as described above - when the part's SerialPart or Batch aspect model is created. But it also contains the Unique IDs of the built-in parts (attributes childItems.catenaXId), as shown in the example above.
+The SingleLevelUsageAsBuilt aspect can only be completed by combining supplier and customer information. Only the supplier will create and update the Usage aspects. The missing information can be provided by the customer(s) with the help of notifications.
 
-For the build-in parts (child items), their Unique ID is not known to the manufacturer initially. Only know are the local ids that are printed on the physical part (serialized part or batch), i.e., manufacturer (BPN), manufacturer part id and serial or batch number. To get the Unique ID of a built-in part, a data provider therefore has to do the following:
+For better understanding, simplified names "parents", "child", "customers" are used in the description.
 
-- Get all necessary local ids for the built-in part:
-  - manufacturer (BPN), manufacturer part id and serial number for serialized parts
-  - manufacturer (BPN), manufacturer part id and batch number for batches
-- The next step is about getting the Unique ID of all built-in parts. There are two ways:
-  - Unique IDs might for built-in parts might already be available locally if Unique ID Push is supported by the data provider and the suppliers of the built-in parts.
-  - Query a supplier's Digital Twin Registry to find the digital twin for this built-in part
+These expressions “parents”, “child” and “customers” can be translated into the attributes of the aspect model.
+
+Translation into SingleLevelUsageAsBuilt aspect:
+
+“parents” -> “catenaXId” in “parentItems”
```

Comment:
I just found out that there was an error in the file, leading to a wrong rendered version. I added the mapping information and corrected the point so that it is more understandable. I will also discuss the point in more details with the team so that we can add more information in support in the future!